components/switch/w-switch.vue

```diff
@@ -16,28 +16,18 @@ const p = defineProps({
 const model = createModel({ props: p });
 const inputEl = ref(null);
 
-const switchClasses = computed(() => [
-  ccSwitch.label,
-  {
-    [ccSwitch.labelDisabled]: p.disabled,
-  },
-]);
+const switchClasses = computed(() => [ccSwitch.base, p.disabled && ccSwitch.disabled]);
 
 const trackClasses = computed(() => [
   ccSwitch.track,
-  {
-    [ccSwitch.trackActive]: model.value && !p.disabled,
-    [ccSwitch.trackInactive]: !model.value && !p.disabled,
-    [ccSwitch.trackDisabled]: p.disabled,
-  },
+  p.disabled && ccSwitch.trackDisabled,
+  model.value && !p.disabled ? ccSwitch.trackActive : ccSwitch.trackInactive,
```

Comment:
Because we check `p.disabled` in line 23, I'd follow the same pattern here for better readability, so `model.value && p.disabled ? ccSwitch.trackInactive : ccSwitch.trackActive`. Wdyt?

Reply:
Hmm but `p.disabled` needs to be false regardless if `model.value` is true or false. 🤔  But I could change it to this instead: `  !p.disabled && (model.value ? ccSwitch.trackActive : ccSwitch.trackInactive)` if that is increases readability? :)

Reply:
I don't understand :D I merely reversed the order after `&&` in line 24 from:
`model.value && !p.disabled ? ccSwitch.trackActive : ccSwitch.trackInactive,`
to 
`model.value && p.disabled ? ccSwitch.trackInactive : ccSwitch.trackActive,`

Reply:
Yeah and I think that I made a mistake when refactoring those conditions. 😄 🙈  

It was like this before (p.disabled is false for both `ccSwitch.trackActive` and `ccSwitch.trackInactive`: 
```````
{
    [ccSwitch.trackActive]: model.value && !p.disabled,
    [ccSwitch.trackInactive]: !model.value && !p.disabled,
    [ccSwitch.trackDisabled]: p.disabled,
  },
```````
So I think it is better to refactor it to this:
` !p.disabled && (model.value ? ccSwitch.trackActive : ccSwitch.trackInactive)`

Since p.disabled is false in both conditions. 🤔

Reply:
Updated in this commit: [351ecf7](https://github.com/warp-ds/vue/pull/187/commits/351ecf79301195750c829e368af8d648b169268b)

Reply:
Ah, now I understand 😄